src/db/conditions/InConditionBuilder.php

```
@@ -179,88 +213,126 @@ protected function buildSubqueryInCondition($operator, $columns, $values, &$para
             return $operator === 'IN' ? '0=1' : '';
         }
 
-        $sqlColumns = [];
-        foreach ($columns as $i => $column) {
-            if ($column instanceof Expression) {
-                $column = $column->expression;
-            }
-            $sqlColumns[] = strpos($column, '(') === false ? $this->queryBuilder->db->quoteColumnName($column) : $column;
-        }
+        return '(' . implode($operator === 'IN' ? ' OR ' : ' AND ', $vss) . ')';
+    }
```

PHPMD (2.15.0)

[warning] The method buildCompositeInCondition() has a Cyclomatic Complexity of 13. The configured cyclomatic complexity threshold is 10. (undefined)

(CyclomaticComplexity)